packages/edit-post/README.md

```diff
@@ -100,6 +100,60 @@ _Returns_
 
 -   `WPElement`: The WPElement to be rendered.
 
+<a name="PluginDocumentSettingPanel" href="#PluginDocumentSettingPanel">#</a> **PluginDocumentSettingPanel**
+
+Renders items below the Status & Availability panel in the Document Sidebar.
+
+_Usage_
+
+```js
+// Using ES5 syntax
+var el = wp.element.createElement;
+var __ = wp.i18n.__;
+var registerPlugin = wp.plugins.registerPlugin;
+var PluginDocumentSettingPanel = wp.editPost.PluginDocumentSettingPanel;
+
+function MyDocumentSettingPlugin() {
+	return el(
+		PluginDocumentSettingPanel,
+	{
+			className: 'my-document-setting-plugin',
+		},
+			__( 'My Document Setting Panel' )
+		);
+	};
+
+	registerPlugin( 'my-document-setting-plugin', {
+	render: MyDocumentSettingPlugin
+	} );
```

Comment:
The spacing in here is a bit awkward. Please don't let that block a merge — I am DYING for this feature — but wanted to point it out in case it's a quick fix :)

Reply:
@chrisvanpatten thanks for mentioning this. I've updated :)